crates/mysten-network/Cargo.toml

```diff
@@ -15,7 +15,7 @@ futures = "0.3.21"
 http = "0.2.8"
 multiaddr = "0.17.0"
 serde = { version = "1.0.140", features = ["derive"] }
-tokio = { workspace = true, features = ["sync", "rt", "macros"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
I'd prefer to stick to just the required features here, I think they are fine-tuned in snarkOS as well

Reply:
this applies to other instances of the `tokio` dep as well

Reply:
yeah, that was a blunt axe 😅